### PR TITLE
feat: add adornments to combobox

### PR DESCRIPTION
**Pull request by poulch**
I want to merge this change because it fixes the selected option when the user clear input and adds adornment to allow create swatch component.

This PR closes #...

### Pull Request Checklist

- [ ] New components are exported from `./src/components/index.ts`.
- [ ] The storybook story is created and documentation is properly generated.
- [ ] New component is wrapped in `forwardRef`.

### CODE REVIEW

File: src/components/Combobox/Static/StaticCombobox.stories.tsx [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 import { Combobox } from "..";
 
 const options = [
-  { value: "color-black", label: "Black" },
-  { value: "color-red", label: "Red" },
-  { value: "color-green", label: "Green" },
-  { value: "color-blue", label: "Blue" },
-  { value: "color-orange", label: "Orange" },
-  { value: "color-purple", label: "Purple" },
+  { value: "black", label: "Black" },
```
Comment (krzysztofzuraw) [inline src/components/Combobox/Static/StaticCombobox.stories.tsx:9]: **Question:** why changing `color-black` to `black`?

It was added to add more difference between value and label - as in real world scenario those values will be some kind of ids etc

File: src/components/ExperimentalFilters/Row.tsx [hunk 2]
```diff
@@ -39,8 +39,8 @@ export const RowComponent = ({
         onChange={(value) => {
           emitter.changeLeftOperator(
             index,
-            value,
-            leftOptions.find((option) => option.value === value.value)?.type
+            value!,
```
Comment (krzysztofzuraw) [inline src/components/ExperimentalFilters/Row.tsx:42]: **Question:** why this change?

Maybe it will be more safe to check if value exist first and then executing `emitter.changeLeftOperator`?

**Review by krzysztofzuraw — APPROVED** [review]

**Review by Droniu — APPROVED** [review]
